### PR TITLE
Restructured install bash script 

### PR DESCRIPTION
**Pull request by AdamSlack**
Install.sh updated to reflect the changes in directory structures.

### CODE REVIEW

File: pipeline/archiver/retriever/retriever.js [hunk 1]
```diff
@@ -72,4 +72,4 @@ async function fetchAppData(searchTerm, numberOfApps, perSecond) {
 // }).then(
 //     (app) => logger.info(app),
 //     (err) => logger.err(err.message)
-// );
\ No newline at end of file
+// );
```
Comment (deanott) [inline pipeline/archiver/retriever/retriever.js]: Remove commented example permissions that is below this code.

File: pipeline/db/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "dependencies": {
+        "pg": "^7.0.2"
+    },
+    "devDependencies": {},
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "author": "",
+    "license": "BSD"
+}
```
Comment (deanott) [inline pipeline/db/package.json]: We do not want most of this of the terms inside package and devDependencies. License is good though. Add author too :)

**Review by deanott — COMMENTED** [review]
Review changes

File: pipeline/archiver/retriever/retriever.js (hunk 1 above)
Comment (AdamSlack) [reply pipeline/archiver/retriever/retriever.js]: ??

File: pipeline/db/package.json (hunk 2 above)
Comment (AdamSlack) [reply pipeline/db/package.json]: what at change do you want making?

Comment (deanott) [reply pipeline/db/package.json]: devDependencies, scripts and add author plus description

File: pipeline/archiver/retriever/retriever.js (hunk 1 above)
Comment (AdamSlack) [reply pipeline/archiver/retriever/retriever.js]: done

File: pipeline/db/package.json (hunk 2 above)
Comment (AdamSlack) [reply pipeline/db/package.json]: Ah, My B.

**Review by deanott — APPROVED** [review]

**Pull request merged**